lib/class-wp-rest-menu-items-controller.php

```diff
@@ -929,6 +999,41 @@ protected function prepare_items_query( $prepared_args = array(), $request = nul
 		return $query_args;
 	}
 
+	/**
+	 * Checks whether current user can assign all terms sent with the current request.
+	 *
+	 * @param WP_REST_Request $request The request object with post and terms data.
+	 *
+	 * @return bool Whether the current user can assign the provided terms.
+	 */
+	protected function check_assign_terms_permission( $request ) {
```

Comment:
What is the difference between this method and the parent implementation?

Reply:
The different is line [1019](https://github.com/WP-API/menus-endpoints/pull/41/files#diff-c2e43307db8225dbe707452b60ff04fcR1019)

```
foreach ( (array) $request[ $base ] as $term_id ) {
```
compared to
```
foreach ( $request[ $base ] as $term_id ) {
``` 

This is because menu is not an array by an int. We can change this in core.